roles/acm_hypershift/README.md

```diff
@@ -45,7 +45,7 @@ At this time the role supports creating the OpenShift required endpoints under t
 | ah_pullsecret_file      | ""                                        | Yes       | Required to pull the hosted cluster release image                                       |
 | ah_release_image        | quay.io/openshift-release-dev/ocp-release | No        | The release image to install, from the specified OCP version                            |
 | ah_node_memory          | 8Gi                                       | No        | Memory which is visible inside the Guest OS (type BinarySI, e.g. 5Gi, 100Mi)            |
-
+| ah_control_plane_availability_policy | HighlyAvailable              | No        | ControllerAvailabilityPolicy specifies the availability policy applied to critical control plane components. Options: HighlyAvailable, SingleReplica |
```

Comment:
I don't see the default value anywhere, could it be defined in defaults?

Reply:
Is not defined in the role because that's what the binary defaults to:
```Shell
$ hcp create cluster --help | grep infra-avail
  --infra-availability-policy string    Availability policy for infrastructure services in guest cluster. Supported options: SingleReplica, HighlyAvailable.
```